tests/test_commandhandler.py

```diff
@@ -397,13 +397,19 @@ def test_edit_prefix(self):
         handler = self.make_default_handler()
         handler.prefix = ['?', '§']
         assert handler._commands == list(combinations(['?', '§'], self.COMMANDS))
+        handler.prefix = ('!', '^')
```

Comment:
new tests must not be necessary

Reply:
They ARE necessary. Test coverage check fails without them.

Reply:
You're missing the point. As I've tried to explain, this PR should not include changes, that make non-signature code changes (and thus new tests) necessary. Specifically: If changing the signatures of `PrefixHandlers` setters requires code-changes to make MyPy happy, don't change them.

Reply:
I've explained it to you before.
Change type hints
-> MyPy fails on third-party calls
-> convert everything to list
-> CodeCov fails
-> add new tests
-> author of the CodeCov and MyPy configs says it wasn't necessary.
It was you who configured these checks. And now you tell me not to pass them.

Reply:
I'm not telling you not to pass the tests. I'm telling you not to change signatures (i.e. change type hints) if that leads to the tests failing.
there is a difference between "not necessary" and "not wanted" ;)

Reply:
The setters were changed in the original PR and it didn't bother you then. Why now?

Reply:
If i roll back all signature changes that require non-signature changes, the PR would be empty.

Reply:
> The setters were changed in the original PR and it didn't bother you then. Why now?

citing my comment on #2152:

> I didn't go over i[t] in detail, yet [...]
>
>* Tuples should only be allowed where it really doesn't matter if it's a tuple or a list [...]

regarding:

> If i roll back all signature changes that require non-signature changes, the PR would be empty.

By now I have the impression that you intentionally try to misunderstand me. I'll try to specify my point one more time:

If you change

```python
def foo(var: List[T]):
```
to 

```python
def foo(var: Union[List[T], Tuple[T]]):
```
and the tests still run with that change, it's fine for this PR and this is what I mean by a "signature change" (because you change the signature of a function).
If the tests fail after that change, please don't include the change in this PR.
If you change something that is *not* the signature of a function and not docstring, I consider that a "non-signature change".

Long story very short and imprecise in a first-impression-comment on a PR with failing tests: "Tuples should only be allowed where it really doesn't matter if it's a tuple or a list" (see above)